### PR TITLE
Added instructions in README.md for installing the dependencies

### PR DESCRIPTION
**Pull request by SHZ66**

### CODE REVIEW

File: tutorial-3.2/README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ This tutorial is meant to demonstrate how to set up, run, and analyze a membrane
 ## Tutorial files
 
 All files necessary for completing the tutorial can be found in this directory.
+Please install required packages before running the tutorial by running the following command:
+```
+conda env create --file=environment.yaml
```
Comment (jeremyleung521) [inline tutorial-3.2/README.md:9]: The file is environment.yml (without the a)

File: tutorial-3.2/README.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ This tutorial is meant to demonstrate how to set up, run, and analyze a membrane
...
+```
+or
+```
+conda env update --name <your WESTPA conda env> --file=environment.yaml
```
Comment (jeremyleung521) [inline tutorial-3.2/README.md:13]: Same as above.

**Review by jeremyleung521 — APPROVED** [review]
Just two typos, but other than that, ready to merge!

File: tutorial-3.2/README.md (hunk 1 above)
Comment (SHZ66) [reply tutorial-3.2/README.md]: Nice catch. Fixed.

**Pull request merged**